.changeset/fluffy-fans-clap.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/client-js': minor
+---
+
+Added Agent.readPlan() for loading submitted plan Markdown.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)